docs/reference/commandline/ps.md

```diff
@@ -51,7 +51,7 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 * ancestor (`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`) - filters containers that were created from the given image or a descendant.
-* isolation (default|hyperv)   (Windows daemon only)
+* isolation (default|process|hyperv)   (Windows daemon only)
```

Comment:
Reading this, I think `default` will be a confusing option for filtering, because it will depend on what the user has set as the default (in the previous PR's, I assumed "default" was the name of the standard isolation mode). Should we drop the "default" option for filtering?